document/core/util/macros.def

```diff
@@ -894,15 +894,17 @@
 
 .. |ok| mathdef:: \mathrel{\mbox{ok}}
 .. |const| mathdef:: \xref{valid/instructions}{valid-constant}{\mathrel{\mbox{const}}}
+.. |catch| mathdef:: \xref{valid/conventions}{valid-labelkind}{\mathrel{\mbox{catch}}}
```

Comment:
This is in a different syntactic class than ok and const, which define (unary) relations. The "catch" marker is just a piece of syntax, like a label or tag. It should be typeset with `\K{}`, and be moved down to the Types section. I'd also probably rename it to LCATCH for that reason.

Reply:
Just noting that the administrative catch (`\CATCHadm`) is also typeset with `\K{}`, like all administrative instruction terminals. 

Would it be clearer if I add a `_{adm}` to the new administrative instructions, or rename this `catch` (now `LCATCH`) to something else? Or is it fine to have `\K{catch}` twice?

Reply:
Seems fine to have it twice, I don't think the two roles can be confused.